lerobot/common/robot_devices/robots/factory.py

```diff
@@ -0,0 +1,44 @@
+def make_robot(name):
+    if name == "koch":
+        from lerobot.common.robot_devices.cameras.opencv import OpenCVCamera
+        from lerobot.common.robot_devices.motors.dynamixel import DynamixelMotorsBus
+        from lerobot.common.robot_devices.robots.koch import KochRobot
+
+        robot = KochRobot(
+            leader_arms={
+                "main": DynamixelMotorsBus(
+                    port="/dev/tty.usbmodem575E0031751",
```

Comment:
add automatic usb port detection ? something like 
```
import os
import time

def list_usb_devices():
    usb_devices = []
    for file in os.listdir('/dev'):
        if file.startswith('tty.usbmodem'):
            usb_devices.append(os.path.join('/dev', file))
    return usb_devices

def detect_active_usb_port(device):
    
    print(f"Please disconnect the {device} and press Enter when done.")
    input()
    initial_devices = list_usb_devices()
    
    print(f"Please reconnect the {device} and press Enter when done.")
    input()

    time.sleep(0.5)
    devices_after_reconnect = list_usb_devices()

    reconnected_device = list(set(devices_after_reconnect) - set(initial_devices))
    usb_port = reconnected_device[0] if reconnected_device else None

    if usb_port:
        return usb_port
    else:
        raise ValueError(f"Could not detect the active USB port of the {device}.")

detect_active_usb_port("Leader Arm Motors Bus")```

Reply:
Genius

Reply:
Added:


    A DynamixelMotorsBus instance requires a port (e.g. `DynamixelMotorsBus(port="/dev/tty.usbmodem575E0031751"`)).
    To find the port, you can run our utility script:
    ```bash
    python lerobot/common/robot_devices/motors/dynamixel.py
    >>> Finding all available ports for the DynamixelMotorsBus.
    >>> ['/dev/tty.usbmodem575E0032081', '/dev/tty.usbmodem575E0031751']
    >>> Remove the usb cable from your DynamixelMotorsBus and press Enter when done.
    >>> The port of this DynamixelMotorsBus is /dev/tty.usbmodem575E0031751.
    >>> Reconnect the usb cable.
    ```
    To find the motor indices, use [DynamixelWizzard2](https://emanual.robotis.com/docs/en/software/dynamixel/dynamixel_wizard2).